runelite-client/src/main/java/net/runelite/client/plugins/hunter/TrapOverlay.java

```diff
@@ -27,15 +27,21 @@
 import java.awt.Color;
 import java.awt.Dimension;
 import java.awt.Graphics2D;
+import java.util.List;
 import java.util.Map;
 import javax.inject.Inject;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
Fix your imports

Reply:
Is the issue here the blank line?

Reply:
Yes, they should all be in the same block without blank lines separating them.